django/applications/catmaid/control/landmarks.py

```diff
@@ -1192,7 +1191,8 @@ def delete(self, request, project_id, landmark_id, location_id):
         remaining_pci = PointClassInstance.objects.filter(point_id=int(location_id))
         if remaining_pci.count() == 0:
             try:
-                can_edit_or_fail(request.user, point.id, 'point')
+                can_edit_or_fail(request.user, point.id, 'point') # FIXME No variable point here to get id attribute. Probably a bad cut'n'paste
+                                                                  # from method above it?
```

Comment:
That's the most likely explanation. Instead of `point.id`, `location_id` should be passed in. It would be reasonable to store the `int` version from L1183 in `location_id` at the top and use it in L1183, L1191 and L1194.

Reply:
Will do - giving it a different name to avoid potential for changing variable's type